Agent/Utilities/NRLogger.m

```diff
@@ -346,7 +357,6 @@ - (NSData*) jsonDictionary:(NSDictionary*)message {
 
 
     NSMutableDictionary *providedAttributes = [message mutableCopy];
-    [providedAttributes removeObjectsForKeys:@[NRLogMessageLevelKey,NRLogMessageFileKey,NRLogMessageLineNumberKey,NRLogMessageMethodKey,NRLogMessageTimestampKey,NRLogMessageMessageKey]];
```

Comment:
The intention of this line is to remove reserved attributes from user provided attributes when passed to NewRelic.logAttributes.

Reply:
Do you still want to remove this line? If so can you explain why?

Reply:
This line doesn't interfere with auto collected logs like I originally thought. Adding it back it.